src/test/java/com/artipie/asto/InMemoryStorageTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.asto;
+
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.asto.memory.InMemoryStorage;
+import io.reactivex.Flowable;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.reactivestreams.FlowAdapters;
+
+/**
+ * Tests for {@link InMemoryStorage}.
+ *
+ * @since 0.1
+ */
+public final class InMemoryStorageTest {
+
+    @Test
+    void shouldSave() {
+        final BlockingStorage storage = new BlockingStorage(this.storage());
+        final byte[] data = "0".getBytes();
+        final Key key = new Key.From("shouldSave");
+        storage.save(key, data);
+        MatcherAssert.assertThat(storage.value(key), Matchers.equalTo(data));
+    }
+
+    @Test
+    void shouldSaveEmpty() throws Exception {
+        final Storage storage = this.storage();
```

Comment:
@olegmoz in the other tests `storage` variable has a `BlockingStorage` type. Can we do the same here?
Because now this is confusing a little bit.

Reply:
@iakunin I didn't use `BlockingStorage` here, because it is impossible to pass empty `Flow.Publisher` as argument to `save`.

Reply:
@olegmoz got it, thanks.